Rakefile

```diff
@@ -48,16 +48,9 @@ task test: 'test:spec'
 task default: :test
 
 begin
-  require 'rubocop/rake_task'
+  require 'voxpupuli/rubocop/rake'
```

Comment:
I'm generally not a fan of something that you require having side effects like defining the task. Is this the wisest plan? I didn't check the implementation yet, but did want to mention it.

Reply:
I thought  it's a nice way to shorten the Rakefile in each gem. `voxpupuli/rubocop/rake` just defines the rubocop rake task. We don't have to use it, but I think it makes sense because the alternative is that we copy any paste the same code into each gem. The code sets 3 rubocop options. When rubocop changes their API and deprecates one of those options, we currently need to touch every Rakefile. With the approach via voxpupuli-rubocop we can reduce this to a single place.

Reply:
I think we can continue with https://github.com/voxpupuli/beaker-vagrant/pull/72 for now. That only switches to the new gem, but doesn't use the rake task.